app/src/main/java/com/example/parkingsystemjava/mvp/presenter/DateDialogPresenter.java

```diff
@@ -0,0 +1,4 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+public class DateDialogPresenter {
+}
```

Comment:
if you dont implement the functionality of the presenter, you should implement the methods of the contracts, and then put a comment to explain this. But the presenter only, its not right

Reply:
done, commented in presenter that now implements the empty contract. all this will be used in the next feature.